apps/i18n/common/en_us.json

```diff
@@ -1185,7 +1185,8 @@
   "schoolCity": "School City",
   "schoolCityTown": "City / Town",
   "schoolCountry": "School Country",
-  "schoolInfoDialogDescription": "Welcome back!  Are you still teaching at {schoolName}?",
+  "schoolInfoDialogDescription": "Welcome back!  Are you still teaching at ",
+  "schoolInfoDialogDescription_schoolName": "{schoolName}?",
```

Comment:
I'm not sure what the convention here is but should this even be something that can be translated?

Reply:
Good point as well!  It would probably break something.  I updated the key.

Reply:
I agree with @bethanyaconnor, which means you should delete, not update `schoolInfoDialogDescriptionSchoolName`, right?  Then you can have `{schoolName}` instead of `{i18n.schoolInfoDialogDescriptionSchoolName({schoolName})}` in your component.

Reply:
The issue I ran into was applying styles to ({schoolName})}.  I couldn't wrap it in a span without creating a new key/value in en_json.

From a previous convo with Maddie:
{i18n.schoolInfoDialogDescription()} {schoolName}
"  this will break for RTL languages, so I think we should update the schoolInfoDialogDescription string to include our schoolName:

schoolInfoDialogDescription: "Are you still at {schoolName}?",
then we can call it like this: i18n. schoolInfoDialogDescription({schoolName})